Makefile

```diff
@@ -205,8 +205,7 @@ endif
 compile-devworkspace-controller:
 	$(eval GO_PACKAGE_PATH ?= $(shell head -n1 go.mod | cut -d " " -f 2))
 	$(eval BUILD_TIME = $(shell date -u '+%Y-%m-%dT%H:%M:%SZ'))
-	$(eval REF = $(shell cat .git/HEAD | sed 's|ref: ||'))
-	$(eval GIT_COMMIT_ID := $(shell cat .git/${REF}))
+	$(eval GIT_COMMIT_ID := $(shell git rev-parse --short=4 HEAD))
```

Comment:
I don't think this will work, since we're not copying over the full `.git` directory, just `.git/refs` and `.git/HEAD`.

This might be even more broken in downstream -- the docker line is `COPY . .` so it won't get the git info at all

Reply:
Also, in a repo like https://github.com/redhat-developer/codeready-workspaces-images/tree/crw-2-rhel-8/codeready-workspaces-devworkspace-controller, we're not getting a meaningful commit -- we'd be tagging the build as whatever commit hash the bigger repo is using.

Reply:
yeah, it didn't work :(